### PR TITLE
fix(addon-commerce): `Money` causes an extra space

### LINKED ISSUE #5373: 🐞 - `Money` causes an extra space
### Playground Link

https://stackblitz.com/edit/angular-pqnusy

### Description

Even without the currency symbol there seems to be an extra space after the sum, check the stackblitz link

### Angular version

_No response_

### Taiga UI version

_No response_

### Which browsers have you used?

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #5373

### CODE REVIEW

File: projects/addon-commerce/components/money/money.template.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
             [textContent]="value | tuiFractionPart: decimal : precision"
         ></span>
     </span>
-    <span *ngIf="alignRight || currencySymbol.length > 1">&nbsp;</span>
+    <span *ngIf="(alignRight && currencySymbol.length) || currencySymbol.length">&nbsp;</span>
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/addon-commerce/components/money/money.template.html:22]: But it's work as expected ((
image

[the remark this answers is not in the document]
Comment (splincode) [reply projects/addon-commerce/components/money/money.template.html]: fixed

File: projects/addon-commerce/components/money/money.template.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
             [textContent]="value | tuiFractionPart: decimal : precision"
         ></span>
     </span>
-    <span *ngIf="alignRight || currencySymbol.length > 1">&nbsp;</span>
+    <span *ngIf="currencySymbol.length">&nbsp;</span>
```
Comment (vladimirpotekhin) [inline projects/addon-commerce/components/money/money.template.html:22]: no, there should not be a space if currency alignLeft and currency is just one symbol

`+$123` but `+MXN{space}123`

so the condition must be alignRight && currencySymbol || currencySymbol.length > 1

Comment (vladimirpotekhin) [reply projects/addon-commerce/components/money/money.template.html:22]: let's also add to unit 

```
it(`if empty string is passed, there is no extra space character`, () => {
            testComponent.currency = ``;
            fixture.detectChanges();

            const textContent = fixture.debugElement.nativeElement.textContent;

            expect(textContent).toEqual(`237`);
        });
```

Comment (waterplea) [reply projects/addon-commerce/components/money/money.template.html:22]: What's the point of checking length on the right side?

Comment (vladimirpotekhin) [reply projects/addon-commerce/components/money/money.template.html:22]: (by design)

Comment (splincode) [reply projects/addon-commerce/components/money/money.template.html]: fixed

**Review by waterplea — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**